### PR TITLE
feat(pom): fix build with jdk17 for default profiles

### PR DESCRIPTION
**Pull request by jackxu2011**
### What is the purpose of the change

when build with jdk17, the test will failed, fix it.

### Brief change log

- add test java opts for jdk17 test

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

**Comment by aiceflower** [comment]
Modifying third party dependencies also requires modifying the content of the known-dependencies.txt file

**Review by peacewong — DISMISSED** [review]
LGTM.

File: pom.xml [hunk 1]
```diff
@@ -487,6 +576,158 @@
         <artifactId>curator-framework</artifactId>
         <version>${curator.version}</version>
       </dependency>
+
+      <dependency>
+        <groupId>org.apache.hbase</groupId>
```
Comment (peacewong) [inline pom.xml:581]: Hbase should not be used?

File: pom.xml [hunk 2]
```diff
@@ -413,6 +466,50 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.codehaus.jackson</groupId>
```
Comment (peacewong) [inline pom.xml:470]: org.codehaus.jackson not used？

File: pom.xml (hunk 1 above)
Comment (peacewong) [reply pom.xml:581]: ok

File: pom.xml (hunk 2 above)
Comment (peacewong) [reply pom.xml:470]: > some dependencies still used, this just unify the version

ok

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**